### ISSUE TITLE
feature request: place HoverTool box at a fixed location

### ISSUE THREAD

**Issue by tommycarstensen** [bokeh/bokeh#6078]
A coworker of mine saw a scatter plot making use of the `HoverTool`. She asked, if the position of the info box could be fixed. Could I kindly make this feature request? Thanks.

**Comment by mattpap** [comment]
This is a duplicate of one tasks in #1189, but as this is a simple yet useful feature on its own and multiple people asked already about this, I will keep this as a separate issue.

**Comment by timsnyder** [comment]
@mattpap, do you have thoughts on how this should be implemented?  I could really use this and if it is simple, maybe I could try to submit a PR?

**Comment by dptriani** [comment]
is this solved?

**Comment by bryevdv** [comment]
> is this solved? 

No, otherwise the issue would be closed.

**Comment by bryevdv** [comment]
@mattpap are there any of the newer tooltip issues that this could be rolled into and closed?

**Comment by bryevdv** [comment]
@mattpap can [tooltips](https://docs.bokeh.org/en/latest/docs/user_guide/interaction/tooltips.html) be attached to plot canvases at arbitrary locations? If so I think this could be closed.

**Comment by mattpap** [comment]
> an tooltips be attached to plot canvases at arbitrary locations?

Yes, but not in a practical way (can't add tooltip directly to a plot; must be root) and not easily in data space. The first part is fixed in PR #13538. The later still needs addressing.
